etna/metrics/functional_metrics.py

```diff
@@ -42,10 +69,18 @@ def mape(y_true: ArrayLike, y_pred: ArrayLike, eps: float = 1e-15) -> float:
 
     y_true_array = y_true_array.clip(eps)
 
-    return np.mean(np.abs((y_true_array - y_pred_array) / y_true_array)) * 100
+    multioutput_enum = FunctionalMetricMultioutput(multioutput)
+    if multioutput_enum is FunctionalMetricMultioutput.joint:
```

Comment:
I ment sth like:
```
axis = multioutput_enum.get_axis()
```

Reply:
I don't find it nice to put this kind of implementation detail into public interface of enum. We could create a separate private function that does this kind of work. What do you think?

Reply:
Ok, let't create separate function